templates/mod/ssl.conf.erb

```diff
@@ -12,7 +12,9 @@
   SSLSessionCacheTimeout <%= @ssl_sessioncachetimeout %>
   <%- if scope.function_versioncmp([@_apache_version, '2.4']) >= 0 -%>
   Mutex <%= @_ssl_mutex %>
+    <%- if @ssl_compression -%>
```

Comment:
this checks for truthiness. if it evaluates to falsey, this option will not be set (to `off`)